src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.java

```diff
@@ -149,36 +164,94 @@
  *   &lt;property name="onCommentFormat" value="resume constant check"/&gt;
  *   &lt;property name="checkFormat" value="ConstantNameCheck"/&gt;
  * &lt;/module&gt;
+ * &lt;module name="MemberName"/&gt;
+ * &lt;module name="ConstantName"/&gt;
+ * &lt;module name="IllegalCatch"/&gt;
  * </pre>
  * <pre>
- * //stop constant check
- * public static final int someConstant; // won't warn here
- * //resume constant check
- * public static final int someConstant; // will warn here as constant's name doesn't match the
- * // pattern "^[A-Z][A-Z0-9]*$"
+ * class InputSuppressionCommentFilter
+ * {
+ *     int VAR1; // violation , Name 'VAR1' must match pattern '^[a-z][a-zA-Z0-9]*$'
```

Comment:
do indenation as +2 , not a +4.
we need more horizontal space.

Reply:
Done.